modules/test/playwright/tests/osb-faro-web/page.spec.ts

```diff
@@ -347,6 +348,96 @@ test(
 	}
 );
 
+test(
+	'Assert the pages list shows a list of pages',
+	{
+		tag: '@Legacy',
+	},
+
+	async ({apiHelpers, page}) => {
+		const pageTitle1 = 'My Page';
```

Comment:
@rafaellajordao, I think `pageTitle1` should be closer its usage. Also, as it is referenced after `pageTitle2` and `pageTitle3` maybe we should consider renaming all three variables according to the order.